contracts/external/KyberNetwork/KyberNetworkProxyInterface.sol

```diff
@@ -3,6 +3,15 @@ pragma solidity 0.4.24;
 
 /// @title Kyber Network interface
 interface KyberNetworkProxyInterface {
+    function getExpectedRate(
+      address src,
+      address dest,
+      uint srcQty
```

Comment:
Unless its an issue, specify `uint256?`.

Reply:
This is in the external folder. Leaving it to match their interface like we did in the other external files